### PR TITLE
Payment methods

### PR DESCRIPTION
**Pull request by W3NDO**
I have not been able to get around to building the response object.Will let you know when I do.

### CODE REVIEW

File: contract/assembly/service/near.service.ts [hunk 1]
```diff
@@ -35,6 +35,38 @@ export default class NearService {
   ): CreditAccount | null {
     return storage.get<CreditAccount>(accountID);
   }
+
+  /**
+   *
```
Comment (malcolmkiano) [inline contract/assembly/service/near.service.ts:40]: Please add a description to this comment

File: contract/assembly/service/near.service.ts [hunk 2]
```diff
@@ -35,6 +35,38 @@ export default class NearService {
...
+  paymentReceived(
+    accountID: string,
+    date: string  //typically in epoch-time
+  ): string {
+    storage.set<string>(accountID+date, "Paid")
+    return  `{${accountID+"-"+date}: "Paid" }`
+  }
+
+  /**
+   *
```
Comment (malcolmkiano) [inline contract/assembly/service/near.service.ts:54]: No description

File: contract/assembly/service/response.ts [hunk 3]
```diff
@@ -0,0 +1,18 @@
+export default class Response{
```
Comment (malcolmkiano) [inline contract/assembly/service/response.ts:1]: We don't really need this as a service

**Review by malcolmkiano — COMMENTED** [review]
Take a look at the comments once you get a chance. The `Response` service can probably be removed altogether

File: contract/assembly/service/near.service.ts (hunk 1 above)
Comment (W3NDO) [reply contract/assembly/service/near.service.ts:40]: WIll do.

File: contract/assembly/service/response.ts (hunk 3 above)
Comment (W3NDO) [reply contract/assembly/service/response.ts:1]: Where do we put it?

Comment (W3NDO) [reply contract/assembly/service/response.ts:1]: Actually never mind. I'll add it in models.

**Comment by W3NDO** [comment]
> 

Removing it with the next PR.

**Pull request closed without merging**